lib/vdi.py

```diff
@@ -47,6 +47,13 @@ def clone(self):
     def readonly(self) -> bool:
         return strtobool(self.param_get("read-only"))
 
+    def size(self) -> int:
+        return int(self.param_get("virtual-size"))
```

Comment:
I prefer that we be more explicit about this kind of method, `virtual_size` seems better to me. Just to differentiate it from `physical_size` which corresponds to the size used on physical storage and `virtual_size` which corresponds to the choice of the maximum usable size by the user.